.sample.env

```diff
@@ -0,0 +1,3 @@
+# example .env.local file; replace values with your own
+NOTION_TOKEN=secret_rrm123ABCabc456DEFdef789GHIghi012JKLjkl345M
+GA_MEASUREMENT_ID=G-123ABC45DE
```

Comment:
I think we need to remove the secrets!

Reply:
They are not real secrets

Reply:
I have put Xs to clarify that these are not real secrets